src/api/GenericApi.ts

```diff
@@ -1,16 +1,21 @@
 import { default as axios } from "axios";
-import { IStudyProgramsAPI } from "../components/AddStudyProgramModal";
 import { API_URL } from "../Constants";
 
-export const studyProgramsEndpoint = "programmes"
-export const subjectEndpoint = "subjects"
-export const coursesEndpoint = "courses"
-export const profileEndpoint = "profile"
-export const studentsEndpoint = "students"
+export const studyProgramsEndpoint = "programmes" as const
```

Comment:
Added `as const` so that in other files the imported type will be "programmes" instead of string